crates/bitwarden-exporters/src/cxf/editable_field.rs

```diff
@@ -0,0 +1,225 @@
+use bitwarden_vault::FieldType;
+use credential_exchange_format::{
+    EditableField, EditableFieldBoolean, EditableFieldConcealedString, EditableFieldString,
+    EditableFieldWifiNetworkSecurityType,
+};
+
+use crate::Field;
+
+/// Helper function to create a Field from any EditableField type
+pub(super) fn create_field<T>(name: String, field: &T) -> Field
+where
+    T: EditableFieldToField,
+{
+    Field {
+        name: Some(name),
+        value: Some(field.field_value()),
+        r#type: field.field_type() as u8,
+        linked_id: None,
+    }
+}
+
+/// Trait to convert EditableField types to Field values and types
+pub(super) trait EditableFieldToField {
+    fn field_value(&self) -> String;
+    fn field_type(&self) -> FieldType;
+}
+
+impl EditableFieldToField for EditableField<EditableFieldString> {
+    fn field_value(&self) -> String {
+        self.value.0.clone()
+    }
+
+    fn field_type(&self) -> FieldType {
+        FieldType::Text
+    }
+}
+
+impl EditableFieldToField for EditableField<EditableFieldConcealedString> {
+    fn field_value(&self) -> String {
+        self.value.0.clone()
+    }
+
+    fn field_type(&self) -> FieldType {
+        FieldType::Hidden
+    }
+}
+
+impl EditableFieldToField for EditableField<EditableFieldBoolean> {
+    fn field_value(&self) -> String {
+        if self.value.0 { "true" } else { "false" }.to_string()
+    }
+
+    fn field_type(&self) -> FieldType {
+        FieldType::Boolean
+    }
+}
+
+impl EditableFieldToField for EditableField<EditableFieldWifiNetworkSecurityType> {
+    fn field_value(&self) -> String {
+        security_type_to_string(&self.value).to_string()
+    }
+
+    fn field_type(&self) -> FieldType {
+        FieldType::Text
+    }
+}
+
+/// Convert WiFi security type enum to human-readable string
+fn security_type_to_string(security_type: &EditableFieldWifiNetworkSecurityType) -> &str {
+    match security_type {
+        EditableFieldWifiNetworkSecurityType::Unsecured => "Unsecured",
+        EditableFieldWifiNetworkSecurityType::WpaPersonal => "WPA Personal",
+        EditableFieldWifiNetworkSecurityType::Wpa2Personal => "WPA2 Personal",
+        EditableFieldWifiNetworkSecurityType::Wpa3Personal => "WPA3 Personal",
+        EditableFieldWifiNetworkSecurityType::Wep => "WEP",
+        EditableFieldWifiNetworkSecurityType::Other(s) => s,
+    }
```

Comment:
This is more of a personal thing, but when matching enums with big names, I like to star import the variants to avoid the repetition:
```rust
    use EditableFieldWifiNetworkSecurityType::*;
    match security_type {
        Unsecured => "Unsecured",
        WpaPersonal => "WPA Personal",
        Wpa2Personal => "WPA2 Personal",
        Wpa3Personal => "WPA3 Personal",
        Wep => "WEP",
        Other(s) => s,
    }
```